sys/vm/vm_page.h

```diff
@@ -201,10 +201,18 @@
 
 #if PAGE_SIZE == 4096
 #define VM_PAGE_BITS_ALL 0xffu
+#if defined(atomic_set_8)
```

Comment:
Uh, no, we should make sure subword atomics are implemented for all pure-capability architectures, and disable masked atomics for purecap so it's a compile-time error.

Reply:
I think we do that now. This was an hotfix that I failed to drop.